robot-server/tests/protocols/test_protocols_router.py

```diff
@@ -222,7 +225,7 @@ async def test_get_protocol_by_id(
         id="protocol-id",
         createdAt=datetime(year=2021, month=1, day=1),
         protocolType=ProtocolType.PYTHON,
-        metadata=Metadata(),
+        metadata=Metadata().parse_obj({"referencedRunIds": []}),
```

Comment:
If we specify `referencedRunIds` as a Metadata param as mentioned in the other comment then this line can also be written as:
```suggestion
        metadata=Metadata(referencedRunIds=[]),
```

Reply:
we can make this change without adding the field to Metadata but mypy would complain about it